Header header

bool[9] CPI
bool[11] CP
bool[25] PS
bool[13] DG
bool[13] DD
